backend/tests/services/test_loop_item_executions.py

```
@@ -3153,11 +3398,10 @@ def test_local_runtime_payload_leaves_model_materialization_to_app(
     assert "api_key" not in str(payload)
     if executor_type == "automation_manager":
         assert f"project_id: {project.id}" in payload["message"]
+        assert "你是看板的 AI 管家，只负责编排，不执行具体任务。" in payload["message"]
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)